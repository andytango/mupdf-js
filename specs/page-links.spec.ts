import {
  forExamplePages,
  loadExampleFile,
  TEST_DPI,
} from "./helpers/example-file";

describe("pageLinks", () => {
  forExamplePages("should get the page links", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.pageLinks(doc, page, TEST_DPI)).toMatchSnapshot();
  });
});
