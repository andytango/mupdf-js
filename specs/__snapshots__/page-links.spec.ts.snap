// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pageLinks Page [1 / 4] - should get the page links 1`] = `""`;

exports[`pageLinks Page [2 / 4] - should get the page links 1`] = `""`;

exports[`pageLinks Page [3 / 4] - should get the page links 1`] = `""`;

exports[`pageLinks Page [4 / 4] - should get the page links 1`] = `
"<area shape="rect" coords="100,100,261,128" href="http://google.com/">
"
`;
